src/v2/providers/https.ts

```diff
@@ -89,12 +90,13 @@ export function onRequest(
     opts = optsOrHandler as HttpsOptions;
   }
 
-  if ('cors' in opts) {
+  if (isDebugFeatureEnabled('enableCors') || 'cors' in opts) {
```

Comment:
should we add a simple test case to this?

Reply:
+1 to make sure we don't accidentally open someone's function up to the world